frontend/src/components/slc-configurate-function/slc-configurate-function.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react';
+import { useAppDispatch, useAppForm } from 'hooks/hooks';
+import { SLCFunctionCreateRequestDto } from 'common/types/types';
+import { DEFAULT_PAYLOAD } from './common/constants';
+import { slcFunctionCreate as CreateSLCFunctionValidationSchema } from 'validation-schemas/validation-schemas';
+import { SLCFunctionConfigurate as SLCFunctionConfigurateActions } from 'store/actions';
+import {
+  AppRoute,
+  ButtonStyle,
+  ButtonType,
+  InputType,
+} from 'common/enums/enums';
+import { Button, Input } from 'components/common/common';
+import { getNameOf } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+const SLCConfigurateFunction: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const { control, errors, handleSubmit } =
+    useAppForm<SLCFunctionCreateRequestDto>({
+      defaultValues: DEFAULT_PAYLOAD,
+      validationSchema: CreateSLCFunctionValidationSchema,
+    });
+
+  const handleFormSubmit = (payload: SLCFunctionCreateRequestDto): void => {
+    dispatch(SLCFunctionConfigurateActions.createFunction(payload));
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <h2 className={styles.title}>
+        SLC - <br />
+        ServerLess Computing
+      </h2>
+      <section className={styles.formWrapper}>
+        <h3 className={styles.formTitle}>Create Function</h3>
+        <form onSubmit={handleSubmit(handleFormSubmit)}>
+          <ul className={styles.inputGroups}>
+            <li className={styles.inputGroup}>
+              <h3 className={styles.inputTitle}>Function name</h3>
+              <div className={styles.inputWrapper}>
+                <Input
+                  type={InputType.TEXT}
+                  label=""
+                  placeholder=""
+                  name={getNameOf<SLCFunctionCreateRequestDto>('name')}
```

Comment:
Should i use here SLCFunctionCreateFormDto?
```suggestion
                  name={getNameOf<SLCFunctionCreateFormDto>('name')}
```